src/core/lib/iomgr/socket_windows.h

```diff
@@ -114,6 +118,12 @@ void grpc_socket_become_ready(grpc_winsocket* winsocket,
    The value is probed once, and cached for the life of the process. */
 int grpc_ipv6_loopback_available(void);
 
+extern DWORD grpc_wsa_socket_flags;
```

Comment:
Why having it both as a global extern and as a getter? We probably only want the getter. In which case, the variable should become static, and renamed to `s_wsa_socket_flags`

Reply:
No hidden catch. No strings attached. Just my inattentiveness.